ReactAndroid/src/main/java/com/facebook/react/views/art/ARTSurfaceViewShadowNode.java

```diff
@@ -33,6 +35,18 @@
 
   private @Nullable Surface mSurface;
 
+  private boolean mIsBackgroundColorSet;
+  private int mBackgroundColor;
+
+  @ReactProp(name = ViewProps.BACKGROUND_COLOR, customType = "Color")
+  public void setBackgroundColor(Integer color) {
+    mIsBackgroundColorSet = (color != null);
+    if (mIsBackgroundColorSet) {
```

Comment:
Switching from a color to null did work correctly before because `drawOutput` checked `mIsBackgroundColorSet` before drawing a background color.

I still implemented your nullable integer suggestion to make the code clearer.